src/utilities/requests.ts

```diff
@@ -86,12 +86,17 @@ export async function reqHasura<T = any>(
       // This is often thrown when a Postgres exception is raised for a Hasura query.
       // @see https://github.com/hasura/graphql-engine/issues/3658
       throw new Error(error?.extensions?.internal?.error?.message ?? error?.message ?? defaultError);
+    } else if (code === 'parse-failed') {
+      if (error?.extensions?.internal?.response?.body?.errors?.length) {
+        const errorMessage = error?.extensions?.internal?.response?.body?.errors[0];
```

Comment:
Will this ever have more than one error that we'd need to show?

Reply:
I have no idea. @mattdailis do you have a sense for this?

Reply:
We may need to do another pass at error messages and formats since it's not clear to me what all the possibilities are

Reply:
Yeah that sounds reasonable. Thanks!